lib/pdksync.rb

```diff
@@ -79,14 +80,13 @@ def self.main(steps: [:clone], args: nil)
       puts '(WARNING) @output_path does not exist, skipping module' unless File.directory?(output_path)
       next unless File.directory?(output_path)
       if steps.include?(:pdk_convert)
-        Dir.chdir(main_path) unless Dir.pwd == main_path
-        exit_status = run_command(output_path, 'pdk convert --force --template-url https://github.com/puppetlabs/pdk-templates')
+        exit_status = run_command(output_path, '/opt/puppetlabs/pdk/bin/pdk convert --force --template-url https://github.com/puppetlabs/pdk-templates')
         print 'converted, '
         next unless exit_status.zero?
       end
       if steps.include?(:pdk_validate)
         Dir.chdir(main_path) unless Dir.pwd == main_path
-        exit_status = run_command(output_path, 'pdk validate -a')
+        exit_status = run_command(output_path, '/opt/puppetlabs/pdk/bin/pdk validate -a')
```

Comment:
It would maybe be safer to check if the end users installation is present in `/opt/puppetlabs/pdk/bin/pdk` and if not just fall back to `pdk` and warn them that they may not be receiving the latest changes?

Reply:
so we only want to make sure we are using the installed version of pdk. not the bundled version.  erratic behaviour happens when using the bundled one.
```
def call_pdk ()
     return full_path if full-path exists
     else
     use bundled version of pdk and warn
end
```
?